content/04.results.md

```diff
@@ -174,12 +175,38 @@ We performed gene set variant analysis (GSVA) for Hallmark cancer gene sets, dem
 
 <!-- TODO: Add immune deconvolution figure 4C information. -->
 
-#### Telomerase activity 
+![**Figure 4. Transcriptomic overview** A, First two dimensions from UMAP of sample transcriptome data.
+Points are colored by `cancer_group` of the samples they represent. B, Heatmap of GSVA scores for Hallmark gene sets with significant differences, with samples ordered by `cancer_group`. C, <!-- TODO: Add description about immune deconvolution image we use as main. -->](images/main/Figure4.png){#fig:Fig4}
+
+
+#### Prediction of _TP53_ oncogenicity and telomerase activity
+To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA gene expression data[@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
```

Comment:
```suggestion
To understand the _TP53_ phenotype in each tumor, we ran a classifier previously trained on TCGA [@doi:10.1016/j.celrep.2018.03.076] to infer _TP53_ inactivation status.
```
because technically it uses gene expression (X) + somatic variants (Y) 🤷‍♂️

Reply:
Oh good catch, thanks